### DISCUSSION TITLE
v11 Update - Pausing community contributions.

### DISCUSSION THREAD

**Discussion by MikeCodesDotNET** [AvaloniaUI/Avalonia#10599] [General]
We have been working hard since the August preview announcement of Avalonia UI v11. V11 is packed with improvements allowing developers to build better apps for even more platforms. The project has seen incredible growth since the release of Avalonia 0.10. This growth has propelled us to become the most popular .NET Foundation community project in terms of community contributions. These contributions are hugely appreciated but require significant time and effort to review. 

To ensure that the next version of Avalonia UI is as reliable and stable as possible, we have decided to pause reviewing pull requests from the community unless the PR resolves critical bugs. We request that you pause submitting PRs until after v11 to avoid creating a significant backlog. 

While we understand this may be disappointing for developers hoping to see their contributions included in the next version of Avalonia UI, this approach will help us ensure the framework is polished and reliable.

We appreciate your patience and support as we work hard to make Avalonia UI v11 the best UI framework for cross-platform .NET development. The upcoming release is a significant improvement over previous versions, and we look forward to sharing a release candidate with you!

If you’re keen to support Avalonia UI, we would appreciate it if you considered [sponsoring](https://github.com/sponsors/AvaloniaUI) the project. Sponsorship enables us to hire more developers who help manage the enormous amount of work required to handle issues, pull requests, bug fixing and more.

**Comment by grokys** [comment]
@tomenscape this doesn't include your changes to use `StyledProperty` where possible, we'd like to get that in before 11.0.

**Reply by SuperJMN** [reply]
That's completely reasonable. Take your time. 11 will be huge!

**Comment by robloo** [comment]
Can I suggest going about this a different way? Have a clear cutoff schedule when we switch over to bug fixes. Most large projects, to coordinate everyone's effort, post a development schedule and if you miss it, you have to target the next release. Broadcasting a sudden cut-off, while understandable, isn't ideal by any means. A notification before it happens seems fair.

I would communicate this more as a feature freeze as well. I'm assuming you still have time to review bug fixes by anyone. v11 still needs a lot of stability work.

Finally, for controls like ColorPicker where I am the main author I would still hope to get in a round of changes before the control is finally released. Feedback is still coming in and there are changes I need to do. I unfortunately just havnt had any time the past 1+ months but that will change in a week or so.

**Comment by workgroupengineering** [comment]
The community when unaddresss on what to do, tends to solve the problems closest to them. If you provide clear guidelines on what needs to be done before Milestone 11 is released, the community will focus on those.

**Reply by rabbitism** [reply]
I agree with you. I'll be nice to have a roadmap to v11, so we can understand what will be included in the final release.

**Reply by robloo** [reply]
This is a good point. Its probably possible to coordinate the commity better and get more help to the core team. There is an 11.0 milestone list and not all are assigned. It might be possible to find an open issue and ask to work on it.

**Comment by ahopper** [comment]
This makes complete sense to me, maybe another announcement when the api won't change and just pure bug fixes are going to happen to encourage testing and lib developers to update.

**Comment by DeafMan1983** [comment]
Can I bundle into NativeAot if I use dotnet 6.0 or greater?

**Reply by timunie** [reply]
completely wrong place to ask such a question. Either start a new Discussion if one that helps you can be found or ask on telegram. 

Here are some links I found: 
- https://github.com/AvaloniaUI/Avalonia/discussions/8130
- https://github.com/AvaloniaUI/Avalonia/discussions/11267
- https://github.com/AvaloniaUI/Avalonia/discussions?discussions_q=NativeAot+
